source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,160 @@
+.. include:: <isonum.txt>
+
+Robot Battery
+=============
+
+The power supply for an FRC\ |reg| robot is a single 12V 18Ah SLA [Sealed Lead Acid] AGM [Absorbent Glass Mat] battery, capable of high momentary currents in excess of 180A when fully charged. The Robot Battery assembly includes the COTS battery, lead cables with contacts, and SB connector.
+
+Battery Safety & Handling
+-------------------------
+
+**DO NOT CARRY A BATTERY BY THE CABLES**, and always avoid pulling by them.
+Pulling on batteries by the cables may loosen, bend, or break the lug or battery tab -- or even tear the entire tab out of the housing!
+
+.. image:: images/robot-battery/broken-terminal.png
+  :alt: One terminal of an FRC battery fully detached from the battery.
+
+Even if it doesn't get broken, the battery will not be able to provide the same amount of current if the :ref:`connectors are loose <docs/hardware/hardware-basics/preemptive-troubleshooting:Check Battery Connections>`.
+
+Dropping the batteries can bend the internal plates, create bulges, or in the worst case - crack the battery case open. While we do use AGM [absorbant glass mat] technology for safety, when a cell is punctured it still will leak some liquid. This is why FIRST recommends teams have a battery spill kit available.
+
+Finally, certain older battery chargers without "maintenance mode" features can *overcharge* the battery, resulting in boiling off some of the battery acid - another reason for a spill kit.
+
+.. image:: images/robot-battery/boiled.jpg
+  :alt: An FRC battery that was boiled by a high current dumb charger.
+
+Damaged batteries should be safely disposed of as soon as possible. All retailers that sell large SLA batteries, like car batteries, should be able to dispose of it for you. They may charge a small fee, or provide a small "core charge refund", depending on your state law.
+
+.. warning:: **DO NOT** attempt to "repair" damaged or non-functional batteries.
+
+Battery Construction & Tools
+-----------------------------
+
+Battery Leads
```

Comment:
I dislike how this plays with the lug section immediately following. 

More subsections? 
Maybe move the "Standard 6AWG..." line into Battery Leads as the subassembly description, then make each of Wire/SB/Lugs into subsections within that? Do we have a 3rd layer of subheadings I can use?

Reply:
Yes from the [style guide](https://docs.wpilib.org/en/latest/docs/contributing/frc-docs/style-guide.html):
![image](https://user-images.githubusercontent.com/15695521/111081820-2645b300-84d3-11eb-88fe-93cae7534cac.png)

Reply:
I added sub-sub-sections... and then jammed a bunch more information in about heavy wire. 

I remembered that the KOP leads are *not* lugged, and left the lugs as their own sub-section.